### DISCUSSION TITLE
What is the timestamp where  URLs for gradient descent and other topics could be found (not found in "extras").

### DISCUSSION THREAD

**Discussion by minertom** [mrdbourke/pytorch-deep-learning#1194] [General]
Sorry to have to ask this since I have run through most of the course and it is indeed great! Somewhere around the eight hour of the course you listed some links where you have listed some other courses. I think Udemy was one of them where some topics of value are discussed as "extra" material. I did look in the "extras" link for this course and I did not find what was listed in the video. Rather than going through the whole video again (I did spend an hour looking but to no avail) where in the video as in, what time stamp, could that list of URLs be found. 

Thank you so much.
Tom
